src/components/chatInput/index.js

```diff
@@ -248,7 +248,7 @@ const ChatInput = (props: Props) => {
         }
       })
       .catch(err => {
-        props.dispatch(addToastWithTimeout('error', err.message));
+        // props.dispatch(addToastWithTimeout('error', err.message));
```

Comment:
Commenting this out for now - I found that there wasn't actually an error in the catch, so `err.message` would crash the client...super weird.